### DISCUSSION TITLE
[newbie] how can i make the Indeterminate progress bar not bouncing back

### DISCUSSION THREAD

**Discussion by mheepandaaa** [saadeghi/daisyui#2388] [Q&A] [answered]
I want the progress loop all the way but not bouncing back or custom the animation🙏

**Comment by flomero** [comment, marked as answer]
you can edit the animation using the `.progress:indeterminate` selector to change the `animation` and `background-position-x` properties. here is a quick example: https://play.tailwindcss.com/0PeAuzDp7C?file=css
